tests/ha/rear_backup.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
```

Comment:
```suggestion
# Copyright (c) 2021 SUSE LLC
```
Let's go to 2021 and let the pandemic behind us! Let's be optimistic :smile:

Reply:
Done!